src/app/info/components/InfoHero.tsx

```diff
@@ -19,6 +20,14 @@ export const InfoHero = () => {
         </h1>
         <p className="text-base font-medium text-gray-4f mt-2 sm:mt-0 ">
```

Comment:
we could probably command-f for `text-gray-4f` and change that to one of the custom colors